code/modules/modular_computers/hardware/scanners/scanner_medical.dm

```diff
@@ -0,0 +1,14 @@
+/obj/item/weapon/computer_hardware/scanner/medical
+	name = "medical scanner module"
+	desc = "A medical scanner module. It can be used to scan patients and display medical information."
+
+/obj/item/weapon/computer_hardware/scanner/medical/do_on_afterattack(mob/user, atom/target, proximity)
+	if(!can_use_scanner(user, target, proximity))
+		return
```

Comment:
Is this scanner completely nonfunctional until baymed?

Reply:
Yes